ovos_number_parser/numbers_mwl.py

```
@@ -660,8 +708,11 @@ def pronounce_fraction_mwl(word: str, scale: Scale = Scale.LONG) -> str:
 
     print("\n--- Testing Cardinal Extraction ---")
     print(f"'un' -> {extract_number_mwl('un')}")
-    print(f"'una' -> {extract_number_mwl('una')}")
+    print(f"'ũa' -> {extract_number_mwl('ũa')}")
     print(f"'bint'i un' ->", extract_number_mwl(" 	bint'i un"))
```

Ruff (0.13.3)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)